extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -81,6 +81,13 @@ export class AzureAccountProviderService implements vscode.Disposable {
 		return this._onEncryptionKeysUpdated;
 	}
 
+	public async getEncryptionKeys(): Promise<CacheEncryptionKeys | undefined> {
+		if (!this._cachePluginProvider) {
+			await this.onDidChangeConfiguration();
+		}
+		return this._cachePluginProvider?.getCacheEncryptionKeys();
```

Comment:
If you're fine assuming that the provider will always be initialized at this point you can at least clean this up to not say it possibly returns undefined

```suggestion
	public async getEncryptionKeys(): Promise<CacheEncryptionKeys> {
		if (!this._cachePluginProvider) {
			await this.onDidChangeConfiguration();
		}
		return this._cachePluginProvider!.getCacheEncryptionKeys();
```